CentOs 7 and gcc 4.9

Please tell me what parameters to put in cmake for building on centos 7 and gcc 4.9 ?

problem is that on the system centos 7 gcc 4.9  :

```
/home/user/.conan/data/xtensor/0.20.10/omaralvarez/public-conan/package/5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9/include/xtensor/xutils.hpp:618:51: error: ‘is_trivially_default_constructible’ in namespace ‘std’ does not name a template type
     using xtrivially_default_constructible = std::is_trivially_default_constructible<T>;

```

--- a/include/xtensor/xutils.hpp
+++ b/include/xtensor/xutils.hpp
@@ -612,7 +612,7 @@ namespace xt
      * xtrivial_default_construct implemenation *
      ********************************************/
 
-#if !defined(__GNUG__) || defined(_LIBCPP_VERSION) || defined(_GLIBCXX_USE_CXX11_ABI)
+#if !defined(__GNUG__) || defined(_LIBCPP_VERSION) || (defined(_GLIBCXX_USE_CXX11_ABI) && _GLIBCXX_USE_CXX11_ABI)
 
     template <class T>
     using xtrivially_default_constructible = std::is_trivially_default_constructible<T>;

